### PR TITLE
Add ability to bypass certificate validation on HA Url

### PR DESCRIPTION
**Pull request by WhimsySpoon**
I have a self-generated cert on my local HA instance which cannot be validated and results in an SSL issue. This adds a new param called `verify_cert`. It defaults to `True` when not specified in `apps.yaml`, but when set to `False`, it will skip validation of the SSL cert on the `ha_url` endpoint.

### CODE REVIEW

**Comment by ericmatte** [comment]
Hi @WhimsySpoon and thanks for the PR!

I did come up with a similar solution for this issue https://github.com/ericmatte/ad-media-lights-sync/issues/47#issuecomment-999768514, using:
```py
fd = urlopen(url, context=ssl._create_unverified_context())
```

I don't know what solution is the best, but yours looks cleaner.

I will test that out when I will have some free time!

File: apps/media_lights_sync/media_lights_sync.py [hunk 1]
```diff
@@ -21,6 +22,7 @@ def initialize(self):
         args = self.args
         self.lights = args["lights"]
         self.ha_url = args.get("ha_url", None)
+        self.verify_cert = args.get("verify_cert", True)
```
Comment (ericmatte) [inline apps/media_lights_sync/media_lights_sync.py:25]: We should add that option in the README https://github.com/ericmatte/ad-media-lights-sync#app-configuration

**Comment by WhimsySpoon** [comment]
Thanks @ericmatte. This PR allows for backwards compatibility (if the variable is provided in the config, it'll default to the current value). Additionally, trusting all certs, regardless of their validity is never a good idea from a security point of view.

If you're happy with my change, I'll be more than happy to update the readme.

**Comment by WhimsySpoon** [comment]
Readme updated!

**Comment by ericmatte** [comment]
All right make sense! Thanks for the update on the readme.

I will test it and after that it will be good to go.

**Review by ericmatte — APPROVED** [review]
@WhimsySpoon works great. Good job!

**Pull request merged**